src/transformers/models/vits/convert_all_checkpoints.py

```diff
@@ -0,0 +1,1167 @@
+# Fill in the variables with the correct repo_id, set push_to_hub = True.
```

Comment:
Not sure we need to have this stay in the model folder and have to keep maintaining it. The conversion script for one model is enough.